### DISCUSSION TITLE
Ouverture du bandeau lorsqu'on a seulement des cookies techniques et matomo

### DISCUSSION THREAD

**Discussion by joel-jjt** [AmauriC/tarteaucitron.js#1013] [Q&A]
Bonjour,

Existe-t'il un paramétrage pour permettre l'affichage du bandeau d'information sur les cookies lorsque le site utilise seulement des cookies techniques et matomo intégré en privacy by design ?

Après de multiples tentatives de configuration, nous n'arrivons pas à provoquer l'affichage d'un bandeau d'alerte et nous pouvons seulement passer par la page de gestion des cookies pour choisir l'interdiction du tracking sur matomo.

![image](https://user-images.githubusercontent.com/118803032/203260624-6ddaeef1-a41d-4328-8504-8b22794dea56.png)

PS: Le site en question n'est pas encore en ligne.

**Comment by AmauriC** [comment]
Bonjour,

Non, quand le bandeau n'est pas nécessaire, ce qui est le cas, inutile d'ennuyer le visiteur en affichant un bandeau pour lui dire que tout est ok ; )

**Comment by joel-jjt** [comment]
Merci pour ce retour. 

Le cookie Matomo est considéré comme un cookie technique ? 
Dans ce cas, pourquoi est-il possible d'interdire les cookies matomo dans l'outil de gestion des cookies ?

**Reply by AmauriC** [reply]
C'est un cookie de mesure d'audience que l'utilisateur doit pouvoir refuser mais il est conforme pour être actif par défaut.
https://www.cnil.fr/fr/cookies-et-autres-traceurs/regles/cookies-solutions-pour-les-outils-de-mesure-daudience

**Reply by OlivierVS31** [reply]
Bonjour @AmauriC , je rebondis sur cette demande car j'ai la même. Si dans matomo nous sommes compliance avec les préco de la cnil pour ne pas avoir à demander le consentement y a t il un moyen via TaC de ne pas laisser l'option de consentement ? En effet, nous avons des utilisateurs qui par défaut clique sur "tout refuser". Et du coup nous perdons de la valeur dans le tracking alors que nous sommes 100% dans les préco de la CNIL pour ne pas avoir besoin de demander le consentement.

![image](https://user-images.githubusercontent.com/80035181/206677005-c28eac93-4e2f-4ccc-a5db-b77b2ab2d34d.png)

**Reply by AmauriC** [reply]
J'ai justement ouvert hier une issue pour activer, comme c'est le cas actuellement avec atinternet, matomo en mode "cnil" si pas de consentement: https://github.com/AmauriC/tarteaucitron.js/issues/1028
J'essai de m'en occuper rapidement ;)

**Reply by OlivierVS31** [reply]
Super news merci :)
